gui/packages/components/tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+  "defaultSeverity": "error",
+  "extends": [
+    "tslint:latest",
+    "tslint-config-prettier"
+  ],
+  "jsRules": {},
+  "rules": {
+    "object-literal-sort-keys": false,
+    "variable-name": [true, "allow-leading-underscore"]
```

Comment:
Allows leading underscore in variable names.

Normally the one should be fine with using `private` variables without prefixing them:

```ts
class A {
  private let prop: boolean = false;
}
```

But that's impossible for variables backing properties:

```ts
class A {
   // 
  private let _prop: boolean = false;

  get prop(): boolean {
    return this._prop;
  }
  set prop(newValue: boolean) {
     this._prop = newValue;
  }
}
```

The one could also rewrite this as:

```ts
class A {
   // 
  private let backingProp: boolean = false;

  get prop(): boolean {
    return this.backingProp;
  }
  set prop(newValue: boolean) {
     this.backingProp = newValue;
  }
}
```

But that is too verbose, underscore fits very well in this specific use case.